CONTRIBUTING.rst

```diff
@@ -177,65 +188,66 @@ a good opportunity to fill in the scientific context of a given notebook - e.g.
 (button "Fork" in the top right corner).
 
 2. Clone the repository locally on your machine
+::
 
-``git clone git@github.com:username/dat_pyinthesky.git``
+   git clone git@github.com:username/jdat_notebooks.git
 
 3. While this sets up ``origin`` to point to your fork, there is currently no connection to the main ``spacetelescope`` "upstream" repository.  So you can point your local clone to the right repository by doing:
-
-``git remote add upstream https://github.com/spacetelescope/dat_pyinthesky.git``
+::
+    git remote add upstream https://github.com/spacetelescope/jdat_notebooks.git
 
 4. Create a new branch where to start the development and move to that branch
-
-``git branch new_notebook_branch_name``
-
-``git checkout new_notebook_branch_name``
+::
+    git branch new_notebook_branch_name
+    git checkout new_notebook_branch_name
 
 5. Create a new folder where to develop the notebook
-
-``cd jdat_notebook``
-
-``mkdir new_notebook_name``
+::
+    cd notebooks
+    mkdir new_notebook_name
 
 6. Now start building your notebook (new_notebook_name.ipynb)!
 
 7. At any point in the development, save your work and push it up to your forked repository. (Important: you must clear the outputs on your notebook using the Jupyter interface before doing an ``add/commit`` like this.)
+::
+    git add new_notebook_name.ipynb
+    git commit -m "Clear message to state the fix or improvement to the notebook"
+    git push origin new_notebook_branch_name
 
-``git add new_notebook_name.ipynb``
-
-``git commit -m "Clear message to state the fix or improvement to the notebook"``
-
-``git push origin new_notebook_branch_name``
-
-(sometimes you have to reset the upstream, so in that case it is ``git push --set-upstream origin new_notebook_branch_name``)
+sometimes you have to reset the upstream, so in that case it is 
+::
+    git push --set-upstream origin new_notebook_branch_name
 
 8. When you are happy with your notebook, double check that you have satisfied the thecnical requirements of the specific status
 of your notebook (see above).
 
-9. Now you can create a Pull Request from the ``spacetelescope/dat_pyinthesky`` repository. You do that
+9. Now you can create a Pull Request from the ``spacetelescope/jdat_notebooks`` repository. You do that
 by clicking on ``New pull request`` on the webpage, then the link ``compare across forks``. Then set the base repository
-to ``spacetelescope/dat_pyinthesky`` and branch ``master`` and the head fork to
-the branch on your personal fork, so repository ``username/dat_pyinthesky`` and branch ``new_notebook_branch_name``. You
+to ``spacetelescope/jdat_notebooks`` and branch ``master`` and the head fork to
+the branch on your personal fork, so repository ``username/jdat_notebooks`` and branch ``new_notebook_branch_name``. You
 set a title and you click on ``Create pull request``.
 
 One of the team members can then merge your Pull Request.
```

Comment:
I think it needs to be clear that at this point the notebook will go for technical and science review and when those are passed it can be merged.